Makefile

```diff
@@ -11,7 +11,8 @@ install: get_vendor_deps
 	@go install ./cmd/tendermint
 
 build:
-	go build -o build/tendermint ./cmd/tendermint
+	go build --ldflags '-extldflags "-static"' \
```

Comment:
this command is mostly used by devs. Do we need these flags for dev build?

this is where we build a release
https://github.com/tendermint/tendermint/blob/6312eb91bee05171d97cecd509be115b2c6b6477/scripts/dist_build.sh#L38